### ISSUE TITLE
corsAllowOrigin dosn't set *

### ISSUE THREAD

**Issue by rubenvdlinde** [api-platform/api-platform#1029]
Also on [stackoverflow](https://stackoverflow.com/questions/54985470/api-platform-corsalloworigin-dosnt-set)

 I currently have `corsAllowOrigin: "*" #to allow all origins` in my helm values file (as per docs). To make sure that the propper headers are bieng set on return values.

Now I would expect (as per docs) that the Access-Control-Allow-Origin: would now go to "*" however it goes to` Access-Control-Allow-Origin: null.` Thats ofcoure a bit anoying becouse it prevent react native apps of accessing the API.

Question here, is this a bug? Am I using the wrong values? Or overlooking something?

Reference api -> http://178.128.136.195/

**Comment by dunglas** [comment]
Do you use Varnish? If so, be sure to upgrade NelmioCors to the latest version, that fixes a cache pollution issue.

**Comment by rubenvdlinde** [comment]
Good point, I checked that and the NelmioCors that is composed into Api Platform is the newest. According to the composer lock:

 ``` JSON
"name": "nelmio/cors-bundle",
            "version": "1.5.5",
            "source": {
                "type": "git",
                "url": "https://github.com/nelmio/NelmioCorsBundle.git",
                "reference": "adabee944e6fe52ee566caf1770a29355b1e8d83"
            },
```

And [the NelmioCors  release chart](https://github.com/nelmio/NelmioCorsBundle/releases)

**Comment by rubenvdlinde** [comment]
Solved it, should have been:

`corsAllowOrigin: ['*'] #to allow all origins` instead of `corsAllowOrigin: "*" #to allow all origins`

**Comment by dunglas** [comment]
Thanks for the feedback! Could you edit the docs with this fix please?
